src/modules/tests/test-data/cat-adi-part2/test-data.cat-adi-part2.selector.ts

```diff
@@ -14,8 +14,18 @@ export const getDrivingFaultCount = (
 export const getManoeuvresADI2 = (data: CatADI2UniqueTypes.TestData) : CatADI2UniqueTypes.Manoeuvres[] =>
   data.manoeuvres;
 
-export const hasManoeuvreBeenCompletedCatADIPart2 = (manoeuvres: CatADI2UniqueTypes.Manoeuvres[]) =>
-(manoeuvres.length > 0);
+export const hasManoeuvreBeenCompletedCatADIPart2 = (manoeuvres: CatADI2UniqueTypes.Manoeuvres[]) => {
+  if (manoeuvres.length < 2) return false;
```

Comment:
Initialising manoeuvres as `manoeuvres: [{}, {}],` means the length will always be 2 no?

Reply:
True for the first pass against the state slice. It guards against a reducer removing one or both of them at some point. It's belt and braces.